app/controllers/course/assessment/submission/submissions_controller.rb

```diff
@@ -62,10 +62,10 @@ def reload_answer
   def publish_all
     graded_submissions = @assessment.submissions.with_graded_state
     if !graded_submissions.empty?
-      graded_submissions.update_all(
-        workflow_state: 'published', publisher_id: current_user.id, published_at: Time.zone.now,
-        updated_at: Time.zone.now, updater_id: current_user.id
-      )
+      graded_submissions.each do |submission|
```

Comment:
Actually how long it takes for publishing like 200 submissions ?  Upsate_all is used because it took too long before